.github/workflows/reportA11y.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  push:
+    branches:
+      - master
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
So looks like if we want this to only run when a PR is merged on master and this gets run on master the code needs to be as follows: 
```
on:
  pull_request:
    types: [closed]
  branches: 
    - master
jobs: 
  test:
    if: github.event.pull_request.merged == 'true'
```

This will run on master when a PR is closed and merged.

Info is from here: 
https://github.community/t5/GitHub-Actions/Run-action-only-when-a-PR-is-merged/m-p/56155#M9769